docs/development/baremetal_setup.md

```diff
@@ -0,0 +1,66 @@
+# Bare metal machine setup with TheRock
+
+## Linux
+
+1. The machine will need AMD GPU drivers. To install on `Ubuntu 24.04`:
+
+   ```bash
+   # Installing the amdgpu-install package
+   sudo apt update
+   wget https://repo.radeon.com/amdgpu-install/6.4/ubuntu/noble/amdgpu-install_6.4.60400-1_all.deb
+   sudo apt install ./amdgpu-install_6.4.60400-1_all.deb
+   sudo apt update
+
+   # Installing the GPU drivers
+   amdgpu-install --usecase=dkms
+   ```
+
+   For other Linux distributions, you can find the commands at [Installation via AMDGPU installer](https://rocm.docs.amd.com/projects/install-on-linux/en/latest/install/amdgpu-install.html#installation)
+
+1. After drivers have been installed, run:
+
+   ```bash
+   sudo modprobe amdgpu
+   sudo usermod -a -G render,video $LOGNAME
+   ```
+
+1. After the GPU drivers have been setup, it's now time to setup `TheRock`. There are a few ways to install it:
+
+   - [Building from source for Ubuntu 24.04](https://github.com/ROCm/TheRock?tab=readme-ov-file#building-from-source)
```

Comment:
Prefer relative links so these links also work from a local git repository outside of github. When using a website generator, relative links can also get parsed by the generator (e.g. https://www.mkdocs.org/user-guide/writing-your-docs/#linking-to-pages)
```suggestion
   - [Building from source](../../README.md#building-from-source)
```